packages/functions/src/coordinator/GetCoordinatorAppointmentsHandler.test.ts

```diff
@@ -144,6 +148,59 @@ test("Valid request returns appointments of the right hospital", async () => {
   );
 });
 
+test("Valid request all returns appointments of the right hospital", async () => {
```

Comment:
This test is great (although I would change the name to include testing HOSPITAL_COORDINATOR).

We also need a similar test for other roles to make sure they behave correctly when calling with "all".

For example, I would expect that SYSTEM_USER would also get the ICHILOV appointment for the same request.

Try to use `test.each()` to avoid code duplication

Reply:
done